src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-infra-hosts.mdx

```diff
@@ -0,0 +1,241 @@
+    scrapers:
+      cpu:
+        metrics:
+          system.cpu.utilization:
+            enabled: true
+      load:
+      memory:
+        metrics:
+          system.memory.utilization:
+            enabled: true
+      disk:
+      filesystem:
+        metrics:
+          system.filesystem.utilization:
+            enabled: true
+      network:
+      paging:
+        metrics:
+          system.paging.utilization:
+            enabled: true
+      processes:
+      process:
+
+processors:
+  memory_limiter:
+    check_interval: 1s
+    limit_mib: 1000
+    spike_limit_mib: 200
+  batch:
+  cumulativetodelta:
+    metrics:
+      - system.network.io
+      - system.disk.operations
+      - system.network.dropped
+      - system.network.packets
+      - process.cpu.time
+  resource:
+    attributes:
+      - key: host.id
+        from_attribute: host.name
+        action: upsert
+  resourcedetection:
+    detectors: [env, system]
+
+exporters:
+  otlp:
+    endpoint: <var>OTLP_ENDPOINT_HERE</var>
+    headers:
+      api-key: <var>YOUR_KEY_HERE</var>
+
+service:
+  pipelines:
+    metrics:
+      receivers: [hostmetrics]
+      processors: [batch, resourcedetection, resource, cumulativetodelta]
+      exporters: [otlp]
+
+  extensions: [health_check]
+```
+
+## Step 3: View your data [#using-ui]
+
+You can view your collector data in a variety of places in the New Relic UI.
+
+### Browsing host data in infrastructure UI [#using-ui]
+
+By using the recommended configuration for the host receiver, you can view data through the standard features in the [Infrastructure UI](/docs/infrastructure/infrastructure-ui-pages/infrastructure-ui-entities#access-new-ui) (New Host UI) experience.
+
+### Query host metrics [#query-host-metrics]
+
+Once metrics are successfully ingested in New Relic, they are available in [data explorer](/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer) and [query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder).
+
+The following NRQL queries show examples to help you explore the metrics you received:
+
+  * Listing number of metric updates ingested by metric name
+
+    ```
+    SELECT count(*) FROM Metric WHERE metricName LIKE 'system.%' FACET metricName LIMIT max
+    ```
+
+  * Querying specific metrics faceted by host
+
+    ```
+    SELECT average(system.disk.operations) FROM Metric FACET host.name TIMESERIES
+    ```
+
+  * Listing dimensions available for a given metric
+
+    ```
+    SELECT keyset() FROM Metric WHERE metricName = 'system.disk.operations' 
+    ```
+
+Learn more about [querying the metric data type](/docs/data-apis/understand-data/metric-data/query-metric-data-type).
+
+## What's next? [#next]
+
+After you set up your collector, check out our [best-practices](/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts/) guide for tips to improve your use of OpenTelemetry and New Relic.
```

Comment:
i don't think there needs to be a dash in "best-practices"

Reply:
@barbnewrelic, thanks! I removed it!